org.metaborg.spoofax.core/src/main/java/org/metaborg/spoofax/core/stratego/StrategoCommon.java

```diff
@@ -112,6 +155,35 @@
             throw handleException(e, runtime, strategy);
         }
     }
+    
+	@Override
+	public IStrategoTerm invoke(HybridInterpreter runtime, IStrategoTerm input, String strategy,
```

Comment:
Add an `@Nullable` to the return type, since this method can return null.

Reply:
Done